src/reducers/spendings.js

```diff
@@ -0,0 +1,36 @@
+import {confirm} from '../functions';
+
+
+export function spendingsReducer(state = [], action) {
+    switch (action.type) {
+
+        case "ADD_SPENDINGS":
+            let newSpendings = {
+                    title: action.text
+                };
+                return [
+                ...state,
+                newSpendings
+            ];
+
+        case "REMOVE_SPENDINGS":
+            if (confirm(action.title)) {
```

Comment:
reducers should not contain side effects, also there may be need in some business logic where you want to remove the spendings without a confirmation. all sorts of interactions with user should stay in UI components and redux will just execute actions and manipulate state

Lets move the `confirm()` call into UI components where `REMOVE_SPENDINGS` action is triggered. Its OK if you need to call this function multiple times on UI to ask for confirmation